ogr/abstract.py

```diff
@@ -210,13 +219,15 @@ def __init__(
         url: str,
         created_at: str,
         tarball_url: str,
+        git_tag: GitTag = None,
```

Comment:
I am not sure about Pagure and GitLab, but in case of GitHub the release is always related to git tag.

```suggestion
        git_tag: GitTag,
```

Please, look on some GitLab and Pagure instance and check the relation between git tag and release.